index.js

```diff
@@ -31,9 +31,10 @@ function sseMiddleware(__, res, next) {
     if (id) {
       res.write(`id: ${id}\n`);
     }
-
+    if (evt) {
```

Comment:
I think since we are in node environment it would be more natural to have non-destructured arguments, pelase.